q2studio/api/workspace.py

```diff
@@ -78,9 +78,13 @@ def get_artifacts():
 @fail_gracefully
 def create_artifact():
     request_body = request.get_json()
+    try:
+        source_format = request_body['source_format']
+    except KeyError:
+        source_format = None
```

Comment:
How about this - this gets us out of the world of "exceptional" programming:

```suggestion
    source_format = request_body.get('source_format')
```

https://docs.python.org/3/library/stdtypes.html#dict.get

Reply:
Ah, but of course!  Apparently this is my first time using a dictionary 🐒

Reply:
![image](https://user-images.githubusercontent.com/11708906/80263102-5d557080-8644-11ea-91ce-2096d0b505f2.png)

Reply:
lol!